sql/moz-fx-data-shared-prod/telemetry_derived/glam_sample_counts_v1/query.sql

```diff
@@ -0,0 +1,184 @@
+WITH all_samples AS (
+  SELECT
+    os,
+    app_version,
+    app_build_id,
+    channel,
+    metric,
+    key,
+    h1.aggregates
+  FROM
+    clients_histogram_aggregates_v1,
```

Comment:
I think you had mentioned that you weren't able to match the sample count numbers with TMO using this table. Did it seem better using this approach?

Reply:
I have validated the query results with single partition `clients_histogram_aggregates` which matched the TMO view result set vs using something like `date_submission` > '2021-08-01'